configs/softmax_config.cfg

```diff
@@ -0,0 +1,25 @@
+# input loading config
+load_input_path: /mnt/efs/mnist/train.csv
+load_input_type: csv # for the criteo kaggle train.csv
+num_classes: 10
+num_features: 784
+limit_cols: 1000
+normalize: 1
+limit_samples: 50000000
+s3_size: 1000
+momentum_beta: 0.9
+use_bias: 1
+opt_method: momentum
```

Comment:
If there arguments here that we don't use for softmax we should remove them. Also, sgd should be the default optimization method.